<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vasu-PC" name="Test" tests="12" failures="1" timestamp="6 Mar 2018 10:43:39 GMT" time="22.356" errors="0">
  <testcase name="appLaunch" time="0.024" classname="testng.HardSoftAssertTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [13] but found [12]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [13] but found [12]
	at testng.HardSoftAssertTest.appLaunch(HardSoftAssertTest.java:16)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- appLaunch -->
  <testcase name="appLogin" time="0.004" classname="testng.HardSoftAssertTest"/>
  <testcase name="first" time="0.004" classname="testng.FirstTestNgTest"/>
  <testcase name="method1" time="0.004" classname="testng.AnnotationTest"/>
  <testcase name="method2" time="0.004" classname="testng.AnnotationTest"/>
  <testcase name="adminLogin" time="10.042" classname="testng.AdminLoginTest"/>
  <testcase name="method1" time="0.007" classname="testng.AnnotationTest1"/>
  <testcase name="method2" time="0.01" classname="testng.AnnotationTest1"/>
  <testcase name="stringCom" time="0.022" classname="testng.ValidationTest"/>
  <testcase name="appLogin" time="2.969" classname="testng.AdminLoginBTAT"/>
  <testcase name="appLogin" time="0.007" classname="testng.MultipleTestNgTest"/>
  <testcase name="appClose" time="0.005" classname="testng.MultipleTestNgTest"/>
</testsuite> <!-- Test -->
